plugins/web/opentelemetry-instrumentation-long-task/src/instrumentation.ts

```diff
@@ -33,6 +34,8 @@ interface TaskAttributionTiming extends PerformanceEntry {
   containerName: string;
 }
 
+type ObserverCallback = (entry: PerformanceLongTaskTiming) => SpanAttributes;
```

Comment:
a common practice for this repo is to have a `types.ts` file and have all types declarations exported from there.
[example](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/web/opentelemetry-instrumentation-user-interaction/src/types.ts)
I suggest to use it here as well